validate_mailerlite.sh

```
@@ -0,0 +1,120 @@
+if [ -z "$MAILERLITE_ACCESS_TOKEN" ]; then
+    echo "❌ ERROR: MAILERLITE_ACCESS_TOKEN environment variable is not set"
+    echo "Please export your Mailerlite API token:"
+    echo "export MAILERLITE_ACCESS_TOKEN=your_token_here"
+    exit 1
+fi
+
+echo "✅ MAILERLITE_ACCESS_TOKEN is set"
+echo ""
+
+# Change to the project root directory
+cd /Users/sayamnasir/Documents/GitHub/appmixer-connectors
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)